migrations/migrations/20210406205811-name-of-migration.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    /*
+      Add altering commands here.
+      Return a promise to correctly handle asynchronicity.
+
+      Example:
+      return queryInterface.createTable('users', { id: Sequelize.INTEGER });
+    */
+      return queryInterface.createTable('user', {
+        id: {
+          type: Sequelize.INTEGER,
+          unique: true,
+          autoIncrement: true,
+          primaryKey: true,
+          allowNull: false
+        },
+        first_name: {
+          type: Sequelize.STRING,
+          allowNull: false
+        },
+        last_name: {
+          type: Sequelize.STRING,
+          allowNull: false
+        },
+        email: {
+          type: Sequelize.STRING,
+          unique: true,
+          allowNull: false
+        },
+        password: {
+          type: Sequelize.STRING,
+          allowNull: false
+        }
```

Comment:
if timestamps fields are necessary add them here too

Reply:
timestamp fields adde to table definition